Backend/.env.example

```
@@ -1,19 +1,46 @@
-CHATBOT_OLLAMA_MODEL=
-CHATBOT_GEMINI_APIKEY=
-CHATBOT_GEMINI_MODEL=
-CHATBOT_OLLAMA_EMBEDDINGMODEL=
-CHATBOT_GEMINI_EMBEDDINGMODEL=
+# ==========================================
+# FILE MẪU: .env.example
+# Đổi tên file này thành .env trước khi chạy
+# ==========================================
+
+# --- Database & Cache ---
+ConnectionStrings__DefaultConnection=your_mongodb_connection_string_here
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The ConnectionStrings__DefaultConnection key should be in uppercase

(LowercaseKey)